### ISSUE TITLE
[BUG] Upload fails with InvalidOperationException

### ISSUE THREAD

**Issue by LukasQ** [thomasloupe/Slackord#105]
Hi there

I'm on windows11 and Slackord:latest and this pops up all the time. Restart or reconnect don't help
It is somehow random. If I retry in some channels, it's again working, others keep failing.

```11:21:50 Gateway     A SlashCommandExecuted handler has thrown an unhandled exception.:
System.InvalidOperationException: Collection was modified; enumeration operation may not execute.
   at System.Collections.Generic.List`1.Enumerator.MoveNext()
   at Slackord.Classes.DiscordBot.PostMessagesToDiscord(UInt64 guildID, SocketInteraction interaction) in D:\GitHub Projects\Slackord\Slackord\Classes\DiscordBot.cs:line 293
   at Slackord.Classes.DiscordBot.SlashCommandHandler(SocketSlashCommand command) in D:\GitHub Projects\Slackord\Slackord\Classes\DiscordBot.cs:line 98
   at Discord.EventExtensions.InvokeAsync[T](AsyncEvent`1 eventHandler, T arg)
   at Discord.WebSocket.DiscordSocketClient.TimeoutWrap(String name, Func`1 action)```

**Comment by LukasQ** [comment]
Could it be the case that the deconstruction of larger channels take ages and, as it says, I try to push these in the ETL phase and thus to early?

That's the log msg while importing:
```
Parsing Users for import...
Successfully parsed 18 users.
Begin parsing JSON data for general with 705 JSON files...
This import appears to be quite large. Reconstructing will take a very long time and the UI may freeze until completed. Please be patient!
Deconstruction/Reconstruction process started...
Exception processing file canvas_in_the_conversation.json: DeconstructMessage(): Error processing property 'editor_id' with value '' in JSON
Completed importing channel general.


Deconstructing 8432 messages for general
```

and later
```
Deconstructing 8432 messages for general
Found 140 threads in general.
Reconstructed 8434 messages for general
ReconstructAsync(): Collection was modified; enumeration operation may not execute.
```

**Comment by LukasQ** [comment]
Yes, it's the case, I looked only to the progress bar, but not to the logs.
See: https://github.com/thomasloupe/Slackord/issues/106

**Comment by thomasloupe** [comment]
Tracking this, thanks very much for the detailed report! Have left this closed and am replying in #106 :)
